content/english/blog/post-oscar-faas-sync-ml-inference.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Using OSCAR as a FaaS platform for synchronous inference of a machine learning model"
+date: 2022-05-16T09:00:00+01:00
+# post image 
+image: "images/blog/post-20220516-1/plants.jpg"
+# post type (regular/featured)
+type: "featured"
+# meta description
+description: "How to use oscar-cli to deploy and use an inference deep learning model synchronously"
+# post draft
+draft: false
+---
+
+<!--- TODO 1. Hablar sobre el modelo del ejemplo -> This example ... --->
+
+This guide aims to show the usage of the OSCAR platform for Machine Learning inference using a pre-trained Convolutional Neural network classification model by DEEP-Hybrid-DataCloud: the [Plants species classifier](https://marketplace.deep-hybrid-datacloud.eu/modules/deep-oc-plants-classification-tf.html), to classify plant pictures by specie. The PREDICT method works with an RGB image as input and returns a JSON with the top 5 predictions of the plant's specie.
+
+<!--- 2. Hablar sobre las invocaciones síncronas (referenciar post con invocaciones asíncronas) --->
+
+The example is going to be focused on synchronous invocations. OSCAR supports two types of invocations:
+
+* Asynchronous: Where users upload files to a bucket which triggers the execution of the corresponding service as a Kubernetes job, which result will be stored in a specified output bucket. An example of this type of invocation can be found on: [Using OSCAR as a FaaS platform for scalable asynchronous inference of a machine learning model](https://oscar.grycap.net/blog/post-oscar-faas-scalable-ml-inference/)
+
+* Synchronous: The user invokes the service by sending the content of the file to be processed and, instead of creating a Kubernetes job per invocation, the HTTP client, or oscar-cli, waits until the execution finishes. To manage this, a serverless backend is required, OSCAR currently supports [Knative](https://knative.dev) and [OpenFaaS](https://www.openfaas.com/). In this method, the result file is returned in the response and, optionally, it could be stored into an output bucket.
+
```

Comment:
Please add here the diagram from https://docs.oscar.grycap.net/images/oscar-sync.png (re-upload it in this repo just in case)